pkg/tests/apis/provisioning/testdata/github-example.yaml

```
@@ -17,6 +17,9 @@ spec:
     owner: grafana
     repository: git-ui-sync-demo
     branch: dummy-branch
-    branchWorkflow: true
+    workflows:
+    - 'pull-request'
+    - 'branch'
+    - 'push'
```

```suggestion
    - 'branch'
    - 'push'
```
Should we also remove PR workflow from here?